### PR TITLE
HW2

### PR DESCRIPTION
**Pull request by orusalov**

### CODE REVIEW

File: rusalovskyi_oleksandr/02/Task1.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+# then anyway decision will be True
+for l in SYMBOLS_CHECK:
+    counts.append(0)
+
+input_string = input('Please input the string to check: ')
+
+# letter by letter comparison
+for letter in input_string.lower():
+
+    for idx,symbol in enumerate(SYMBOLS_CHECK):
```
Comment (medviediev) [inline rusalovskyi_oleksandr/02/Task1.py]: Space after comma

File: rusalovskyi_oleksandr/02/Task1.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+# make counts list length the same as length of SYMBOLS_CHECK. Add "0" cause if no symbols found,
+# then anyway decision will be True
+for l in SYMBOLS_CHECK:
+    counts.append(0)
+
+input_string = input('Please input the string to check: ')
+
+# letter by letter comparison
+for letter in input_string.lower():
+
```
Comment (medviediev) [inline rusalovskyi_oleksandr/02/Task1.py]: if letter in SYMBOLS_CHECK will prevent unnecessary iterations

File: rusalovskyi_oleksandr/02/Task1.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        if symbol == letter:
+            # counts length the same as SYMBOLS_CHECK
+            counts[idx] += 1
+
+result = True
+
+# counts comparison
+for i in range(1,len(counts)):
+    result = result and counts[i-1] == counts[i]
```
Comment (medviediev) [inline rusalovskyi_oleksandr/02/Task1.py]: It you are trying to implement universal approach you should think about what will happen if you will have more then 2 symbols to check

Comment (orusalov) [reply rusalovskyi_oleksandr/02/Task1.py]: I've tested it on 3 and 4 lowercase letters. If all 3/4 symbols have the same count inside input_string then True.

**Pull request merged**